ui/src/pages/backoffice/Acquisition/Vendor/VendorEditor/components/VendorForm/VendorForm.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import pick from 'lodash/pick';
+import { StringField, TextField } from '@forms';
+import { vendor as vendorApi } from '@api/acquisition/vendor';
+import { AcquisitionRoutes } from '@routes/urls';
+import { goTo } from '@history';
+import { BaseForm } from '@forms';
+
+export class VendorForm extends Component {
+  prepareData = data => {
+    return pick(data, ['name', 'address', 'email', 'phone', 'notes']);
```

Comment:
I would say that the VendorForm just renders the data it receives and any modifications to happen `VendorEditor`. But why do we need to pick only these fields in the first place?If it's about to restrict some fields from being submitted, until now we were using the prop `submitSerializer` which was stripping some fields before submitting...

Reply:
Not everywhere, I checked the wrong example probably `LocationForm.js` is setup in the same fashion and happens to have the same fields 🎉